### PR TITLE
adding sass

### PR DESCRIPTION
**Pull request by yiyingzz**
added sass & set up styles folder & setup snippet

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            "format": "expanded",
+            "extensionName": ".css",
+            "savePath": "/src/styles"
+        }
+    ],
+    "liveSassCompile.settings.excludeList": [
+        "**/node_modules/**",
+        ".vscode/**"
+    ],
+    "liveSassCompile.settings.generateMap": false,
```
Comment (andalexander) [inline .vscode/settings.json:13]: This part looks great!

**Review by andalexander — APPROVED** [review]
Awesome! 🎳

**Review by andrearose26 — APPROVED** [review]
looks good!

**Review by VipinKirthane — APPROVED** [review]
Go ahead please

**Pull request merged**